pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -337,3 +349,61 @@ class DualRK45PseudoStepper(DualRKVdH2RPseudoStepper):
     @property
     def _pseudo_stepper_order(self):
         return 4
+
+
+class DualDenseRKPseudoStepper(BaseDualPseudoStepper):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Does anything higher up the chain expect .a and .b to exist?  If so we may want to load the scheme first.

Reply:
self.a and self.b are not used higher up. Should be fine.

Reply:
I think that's fine for the moment but later we might need to move them up when we start consider error-estimating schemes.